apps/studio/components/interfaces/Organization/BillingSettings/CreditBalance.tsx

```diff
@@ -55,29 +54,30 @@ const CreditBalance = () => {
           <NoPermission resourceText="view this organization's credits" />
         ) : (
           <>
-            {isLoading && (
-              <div className="space-y-2">
-                <ShimmeringLoader />
-                <ShimmeringLoader className="w-3/4" />
-                <ShimmeringLoader className="w-1/2" />
-              </div>
-            )}
+            <FormPanel footer={<CreditTopUp slug={slug} />}>
```

Comment:
tiny nit: extra bottom border when feature flag is off
![image](https://github.com/user-attachments/assets/f4dacc90-514c-4cf7-b318-679249c70d04)